metabase/metabase
# Add syncing modal to admin

Epic: #19440

Adds the modal from https://github.com/metabase/metabase/pull/19447 to Admin.

<img width="851" alt="Screenshot 2021-12-24 at 13 23 47" src="https://user-images.githubusercontent.com/8542534/147345756-c7f95f13-22ea-478b-b761-582a38f1e2f0.png">

How to test:
- Run a fresh metabase instance without a non-sample database added
- Add a new database in Admin. You can copy connection string from Sample Dataset.
- You should see the modal above.
